src/module.c

```diff
@@ -267,6 +268,10 @@ typedef struct RedisModuleBlockedClient {
 static pthread_mutex_t moduleUnblockedClientsMutex = PTHREAD_MUTEX_INITIALIZER;
 static list *moduleUnblockedClients;
 
+static pthread_mutex_t moduleTempClientsMutex = PTHREAD_MUTEX_INITIALIZER;
+static client *moduleTempClients[1024];
```

Comment:
Is it possible to define 1024 as a constant, so that there is no need to calculate its size in `moduleReleaseTempClient`?

Reply:
Sure, I'll do. Actually, compilers calculate it on compile time as it's all about compile time constants. I relied on this but maybe it's better to be explicit.